src/Management/src/Endpoint/ManagementWebHostBuilderExtensions.cs

```diff
@@ -315,4 +322,39 @@ public static IWebHostBuilder AddWavefrontMetrics(this IWebHostBuilder hostBuild
     {
         return hostBuilder.ConfigureServices((_, collection) => collection.AddWavefrontMetrics());
     }
+
+    internal static void ConfigureManagementUrls(this IWebHostBuilder webHostBuilder, List<string> urls)
+    {
+        string managementPort = webHostBuilder.GetSetting(ManagementPortKey);
```

Comment:
Are there scenarios where this method of getting these settings do work? When would this work as opposed to looking for an appsettings.json file directly?

Reply:
When using the WebApplication Builder and also when running tests,  these settings are  available. Specifically when testing, it keeps us from having to add an appsettings.json.
We need the appsettings.json when using the generic hostbuilder.

Reply:
This particular situation has not changed, @dtillman what are your thoughts? If `builder.GetSetting` worked reliably then I'd be fine with it, but what happens next is a fallback that manually looks for `appsettings.json` which is not going to be able to build the same view of the app's config as a developer might expect. 

This discussion item has become disconnected from the file, which is [here](https://github.com/SteeltoeOSS/Steeltoe/pull/1032/files#diff-f0f43afe1e468a0f4913c7ac4ee966246fa6691fd5b11e51731d5b7a42c3491bR328)

Reply:
I'll dig into this next .. finishing up the code for the last RabbitMQ tutorial